### PR TITLE
[TRANSLATION][UTILMAN] Translations for UTILMAN and some other stuff

### PR DESCRIPTION
**Pull request by shytzedaka**
## Purpose

-Translated UTILMAN To hebrew
-Fixed the Hebrew translation a bit in shortcut.inf
- Changed (C) to © cus why not.

### CODE REVIEW

File: media/inf/shortcuts.inf [hunk 1]
```diff
@@ -135,7 +135,7 @@ WINMINE_DESC=WineMine
 SPIDER_TITLE=Spider Solitaire
 SPIDER_DESC=Spider Solitaire
 UTILMAN_TITLE=Accessibility Utility Manager
-UTILMAN_DESC=Launch Accessibility Utility Manager
+UTILMAN_DESC=Launch Accessibility Utility Manage
```
Comment (Fraizeraust) [inline media/inf/shortcuts.inf]: Why? What does this change serve?

Comment (shytzedaka) [reply media/inf/shortcuts.inf]: This is a mistake. i will fix this.

**Comment by HBelusca** [comment]
Generic remark when citing "Copyright ..."
If you use "Copyright 19xx John Doe", the (C) is unnecessary because it also means "Copyright". So having "Copyright (C) xxx" would mean "Copyright Copyright xxx" aka. WTF?!
Therefore, either use "Copyright 19xx John Doe" or "(C) 19xx John Doe", but not both together.

**Comment by shytzedaka** [comment]
> Generic remark when citing "Copyright ..."
> If you use "Copyright 19xx John Doe", the (C) is unnecessary because it also means "Copyright". So having "Copyright (C) xxx" would mean "Copyright Copyright xxx" aka. WTF?!
> Therefore, either use "Copyright 19xx John Doe" or "(C) 19xx John Doe", but not both together.

@Fraizeraust What do you say about this?

**Comment by Fraizeraust** [comment]
I agree with Hermes. Remove the copyright symbol and keep only the word.

**Comment by shytzedaka** [comment]
> I agree with Hermes. Remove the copyright symbol and keep only the word.

Done.
Ready for merging

**Pull request merged**